### DISCUSSION TITLE
Coversations with ChatGPT about genetic algorithm's application in trading

### DISCUSSION THREAD

**Discussion by dinohsu1019** [anyoptimization/pymoo#552] [Show and tell]
I have a long conversation with ChatGPT about genetic algorithm's application in trading:
[my conversation with ChatGPT](https://chat.openai.com/share/ac43ac74-4c32-42f7-84ce-9ed634080d1f)

The original idea comes from a Medium article and now there are many. I have been doing it with pymoo since Dec. 2023, and there are many things to discuss, I'd like to talk with people who share the same interest here, please join my discussion about this fascinating topic.

Thanks in advance.

**Comment by languagelawyer** [comment]
I decided to start a separate topic https://github.com/anyoptimization/pymoo/discussions/711
